scripts/scil_tractogram_dpp_math.py

```diff
@@ -65,12 +64,12 @@ def _build_arg_parser():
                    'streamline. Set to dpp if the operation is to be \n'
                    'performed on each point separately resulting in a \n'
                    'single value per point.')
-    p.add_argument('--in_dpp_name',  nargs='+', required=True,
+    p.add_argument('--in_dpp_name',  nargs='+', required=True, metavar='key',
                    help='Name or list of names of the data_per_point for \n'
                         'operation to be performed on. If more than one dpp \n'
                         'is selected, the same operation will be applied \n'
                         'separately to each one.')
-    p.add_argument('--out_name', nargs='+', required=True,
+    p.add_argument('--out_name', nargs='+', required=True, metavar='key',
```

Comment:
same metavar ?? no need for this one. I also only use metavar if used in the main but maybe it makes the help prettier.

Reply:
The metavar is never used in the main. (Maybe you think about dest=?). It's for the help. If you don't put it, it prints:
`  --out_name OUT_NAME [OUT_NAME ...]`

Now it prints:
`--out_name key [key ...]`

I found it less visually... invasive... but I can revert if you want.

Reply:
oupsi my bad yes I mixed these two.